etl/steps/data/garden/countries/2023-09-25/gleditsch.meta.yml

```diff
@@ -0,0 +1,20 @@
+# NOTE: To learn more about the fields, hover over their names.
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/en/latest/architecture/metadata/reference/dataset/
```

Comment:
Both links fail here

Reply:
Being fixed in https://github.com/owid/etl/pull/1694